apps/desktop-wallet/src/api/apiDataHooks/address/useFetchAddressInfiniteTransactions.ts

```diff
@@ -31,18 +32,13 @@ interface UseFetchAddressInfiniteTransactionsProps {
 const useFetchAddressInfiniteTransactions = ({ addressHash }: UseFetchAddressInfiniteTransactionsProps) => {
   const networkId = useAppSelector((s) => s.network.settings.networkId)
 
-  const [fetchedTransactionListAt, setFetchedTransactionListAt] = useState(0)
-  const refresh = useCallback(() => setFetchedTransactionListAt(new Date().getTime()), [])
-
   const { data: addressLatestTx, isLoading: isLoadingLatestTx } = useFetchAddressLatestTransaction({ addressHash })
-  const { data, fetchNextPage, isLoading, hasNextPage, isFetchingNextPage } = useInfiniteQuery(
-    addressTransactionsInfiniteQuery({
-      addressHash,
-      timestamp: fetchedTransactionListAt,
-      networkId,
-      skip: isLoadingLatestTx
-    })
-  )
+
+  const query = addressTransactionsInfiniteQuery({ addressHash, networkId, skip: isLoadingLatestTx })
+
+  const { data, fetchNextPage, isLoading, hasNextPage, isFetchingNextPage } = useInfiniteQuery(query)
+
+  const refresh = useCallback(() => queryClient.refetchQueries({ queryKey: query.queryKey }), [query.queryKey])
```

Comment:
Instead of refreching by generating a new `timestamp`, we are now prefetching the proper way.